config_template.cfg

```diff
@@ -1293,7 +1550,111 @@ LEVELS_TIME_ACCURATE_LTS= 1
 %
 % Specify the method for matrix coloring for Jacobian computations (GREEDY_COLORING, NATURAL_COLORING)
 KIND_MATRIX_COLORING= GREEDY_COLORING
+%
+% Specify shock capturing method for DG
+KIND_FEM_DG_SHOCK= NONE
+%
+% Time discretization
+TIME_DISCRE_FEA= NEWMARK_IMPLICIT
+%
+% Parameter alpha for Newmark scheme (s)
+NEWMARK_BETA= 0.25
+%
+% Parameter gamma for Newmark scheme (s)
+NEWMARK_GAMMA= 0.50
+%
+% Newmark - Generalized alpha - coefficients
+TIME_INT_STRUCT_COEFFS= 0.0
+%
+% Apply dead loads
+INCREMENTAL_LOAD= NO
+%
+% Maximum number of increments of the
+NUMBER_INCREMENTS= 10
+%
+% Definition of the  UTOL RTOL ETOL
+INCREMENTAL_CRITERIA= (0.0, 0.0, 0.0)
+%
+% Use of predictor
+PREDICTOR= NO
+%
+% Order of the predictor
+PREDICTOR_ORDER= 0
+
+% ------------------- TOPOLOGY OPTIMIZATION --------------------%
+TOPOLOGY_OPTIMIZATION= NO
+%
+%
+TOPOL_OPTIM_OUTFILE= element_derivatives.dat
+%
+%
+TOPOL_OPTIM_SIMP_EXPONENT= 1.0
+%
+%
+TOPOL_OPTIM_SIMP_MINSTIFF= 0.001
+%
+%
+TOPOL_OPTIM_FILTER_KERNEL= (DILATE, ERODE)
+%
+%
+TOPOL_OPTIM_FILTER_RADIUS= 0.0
+%
+%
+TOPOL_OPTIM_KERNEL_PARAM= 0.01
+%
+%
+TOPOL_OPTIM_SEARCH_LIMIT= 0
+%
+%
+TOPOL_OPTIM_PROJECTION_TYPE= NO_PROJECTION
+%
+%
+TOPOL_OPTIM_PROJECTION_PARAM= 0.0
```

Comment:
I prefer that config_template.cfg lists all available options so it's the first place for non-developers to look for existing keywords/functionality. I have copied the info from the example into config_template.

Reply:
A 2500-line file is not helpful for anyone. By listing the main option and pointing to the detailed example you make it easier to look for existing features, then if the user is actually interested in that feature they have a concise example of how to use it, the example has the flexibility of explaining what other options are important for a given feature, which is not very easy to do with the template.
Also, I don't want people to copy this massive file to start a new case, it doesn't make sense.

Reply:
We can also just delete the entire file and put all the information on the website. Users can use the testcase examples instead. This would be preferable in my opinion, but it's is a little more work to move the info to the website and add links to relevant testcases. This PR should be an intermediate solution imo.

Reply:
@pcarruscag my idea is to put *all* available options in config_templace.cfg. If it's OK with you, I prefer to keep these TOPOL options in there as well.